User Types API deprecation (#5084)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

## User Schema operations

> Each of the operations described here affects the Schema associated with a single [user type](/docs/reference/api/user-types). The `${typeId}` element in the URL specifies which type. It can be the literal `default` to operate on the Schema of the default user type, which is created when the org is initialized, or it can be a schema ID.

Reviewer: ```suggestion
Each of the operations described here affects the schema that's associated with a single [user type](/docs/reference/api/user-types). The `${typeId}` element in the URL specifies which type. It can be the literal `default` to operate on the schema of the default user type, which is created when the org is initialized, or it can be a schema ID.
```

> Each user type has an associated Schema. In the future, the link between Schema and user type may be extended (for example, to allow multiple Types to share a Schema) but for now this is a 1:1 relationship. You can obtain the schema ID for the Schema associated with a [user type](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserType/#tag/UserType) object from its `schema` link. The `schema` link is also included in individual [User](/docs/reference/api/users/#user-object) objects.

Reviewer: ```suggestion
Each user type has an associated schema. In the future, the link between schema and user type may be extended (for example, to allow multiple types to share a schema) but for now this is a 1:1 relationship. You can obtain the schema ID for the schema that's associated with a [user type](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserType/#tag/UserType) object from its `schema` link. The `schema` link is also included in individual [User](/docs/reference/api/users/#user-object) objects.
```

Each of the operations described here affects the Schema associated with a single [User Type](/docs/reference/api/user-types). The `${typeId}` element in the URL specifies which type. It can be the literal `default` to operate on the Schema of the default User Type, which is created when the org is initialized, or it can be a schema ID.

> Each User Type has an associated Schema. In the future, the link between Schema and User Type may be extended (for example, to allow multiple Types to share a Schema) but for now this is a 1:1 relationship. You can obtain the schema ID for the Schema associated with a [User Type](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserType/#tag/UserType) object from its `schema` link. The `schema` link is also included in individual [User](/docs/reference/api/users/#user-object) objects.

Reviewer: Do User Type, Schema, and Types need to be capitalized?

## App User Schema operations

> The [user types](/docs/reference/api/user-types) feature doesn't extend to applications. All users assigned to a given application use the same [App User Schema](#app-user-schema-object). Thus, unlike the User Schema operations, the App User Schema operations all specify `default` and don't accept a Schema ID.

Reviewer: ```suggestion
The [user types](/docs/reference/api/user-types) feature doesn't extend to apps. All users assigned to a given app use the same [app user schema](#app-user-schema-object). Thus, unlike the user schema operations, the app user schema operations all specify `default` and don't accept a schema ID.
```

## Group Schema operations

> The [user types](/docs/reference/api/user-types) feature doesn't extend to groups. All groups use the same [Group Schema](#group-schema-object). Unlike User Schema operations, Group Schema operations all specify `default` and don't accept a Schema ID.

Reviewer: ```suggestion
The [user types](/docs/reference/api/user-types) feature doesn't extend to groups. All groups use the same [group schema](#group-schema-object). Unlike user schema operations, group schema operations all specify `default` and don't accept a schema ID.
```
